src/BrewingRecipes.cpp

```diff
@@ -55,7 +55,7 @@ void cBrewingRecipes::ReloadRecipes(void)
 		AddRecipeFromLine(ParsingLine, LineNum);
 	}  // while (getline(ParsingLine))
 
-	LOG("Loaded " SIZE_T_FMT " brewing recipes", m_Recipes.size());
+	LOG("Loaded %zu brewing recipes", m_Recipes.size());
```

Comment:
%zu, not {}?

Reply:
`LOG` still uses the `printf` style formatting to save having to change every single log call.  It could still be changed to `FLOG` though, in which case `{}` would be correct.

Reply:
Okay, fair. As long as it doesn't break MSVC (I don't have MSVC, so I can't test it).

Reply:
It uses `fmt::sprintf` which doesn't rely on the standard library so it's fully portable.